client/app/components/composites/ListingCardPanel/ListingCardPanel.story.js

```diff
@@ -0,0 +1,63 @@
+import r from 'r-dom';
+import { mount } from 'enzyme';
+
+import { storify } from '../../Styleguide/withProps';
+import { toFixedNumber } from '../../../utils/numbers';
+
+import ListingCardPanel from '../ListingCardPanel/ListingCardPanel';
+import ListingCard from '../ListingCard/ListingCard';
+
+const LISTINGS_COUNT = 24;
+
+const { storiesOf, specs, expect } = storybookFacade;
+const containerStyle = { style: { background: 'white' } };
+
+
+const listingCardTemplate = (title, perUnit, price, distance) => (
+  r(ListingCard, Object.assign({},
```

Comment:
Actually it is not. (We have stories with default props merged with overwritten props - this was copy-paste blindness...) I'll remove.